src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -153,6 +153,36 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
     return this.isSamsungInternet();
   }
 
+  requiresContextRecreationForAudioWorklet(): boolean {
+    // Definitely not Chrome; no worries.
+    if (!('chrome' in global)) {
+      return false;
+    }
+
+    const mac = navigator.userAgent.match(/Macintosh/) !== null;
```

Comment:
Isn't it simpler to use the detect-browser dependency this.browser.os?

Reply:
`detect-browser` seems to be wrong more often than it's right, so I chose to be very specific here instead.